backend/django_core/static_src/components/page/explore/Index.svelte

```diff
@@ -0,0 +1,220 @@
+
+                <div class={cn(klass, "group dropdown dropdown-bottom")}>
+                    <span class="font-semibold leading-none md:text-[1vw]">{title}</span>
+                    <div class="relative flex items-center">
+                        <span class="absolute flex items-center md:gap-[0.25vw] cursor-pointer opacity-100 duration-300 group-focus-within:opacity-0">
+                            {#if selected_items.length > 0}
+                                <span class="ml-3 badge badge-primary rounded p-1 text-sm font-semibold md:ml-[0.75vw] md:rounded-[0.25vw] md:p-[0.35vw] md:text-[0.85vw] md:h-[1.5vw]">
+                                    <!-- show first item -->
+                                    {selected_items[0][1]}
+                                </span>
+                                <!-- show count of remaining items if exists -->
+                                {#if selected_items.length > 1}
+                                    <span class="ml-1 rounded badge md:h-[1.5vw] p-1 text-sm font-semibold md:ml-[0.15vw] md:rounded-[0.25vw] md:p-[0.35vw] md:text-[0.85vw]">
+                                        +{selected_items.filter((item) => item !== selected_items[0]).length}
+                                    </span>
+                                {/if}
+                            {:else}
+                                <span class="ml-3 text-base md:ml-[1vw] md:text-[0.9vw]">Any</span>
+                            {/if}
+                        </span>
+                        <input
+                            bind:value={option[1].value}
+                            on:blur={() => (option[1].value = "")}
+                            type="text"
+                            tabindex="0" role="button"
+                            class="peer w-full rounded-lg border-none bg-neutral py-3 text-base leading-none placeholder focus:ring-0 md:w-[11vw] md:rounded-[0.5vw] md:bg-neutral text-neutral-content md:py-[0.8vw] md:pl-[1vw] md:text-[1vw] placeholder:font-medium font-semibold"
+                        />
+                        {#if selected_items.length > 0}
+                            <button
+                                on:click={() => clear_selected_items(option[0])}
+                                class="btn !bg-transparent border-none absolute right-0 mr-3 w-4 p-0 md:mr-[1vw] md:w-[1vw]"
+                            >
+                                <coreproject-icon-cross class="md:w-[1vw]"></coreproject-icon-cross>
+                            </button>
+                        {:else}
+                            <button class="btn !bg-transparent border-none absolute right-0 mr-3 w-4 p-0 md:mr-[1vw] md:w-[1vw]">
+                                <coreproject-icon-chevron class="md:w-[1vw]"></coreproject-icon-chevron>
+                            </button>
+                        {/if}
+                    </div>
+                    <!-- svelte-ignore a11y-no-noninteractive-tabindex -->
```

Comment:
This should not be the case. Do you need to make it like this?

Reply:
Yes? cause without `tabindex` this dropdown will close after clicking ( which we dont want like in v3 ).
OR we can use a clickable element like `button` that looks off :\

Reply:
> we can use a clickable element like button that looks off :\

This is what i did